.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
+before_install:
+    - git clone https://github.com/nroduit/Weasis.git
+    - cd Weasis 
+    - git checkout tags/3.0.4
```

Comment:
new version of weasis may contain some changes which break build of extension
so we have two options:
1.  build latest version and fix incompatibilities if build crashes with new release
2.  with new release of weasis test it and fix if some of extension's dependency changed, after this increment version

Reply:
Yeah option 2 makes sense - but because we are only testing with this version we should update our Readme to tell users to download 3.0.4 version of Weasis to use this plugin.

Reply:
done